schema@v1.1.1/dataset.json

```diff
@@ -7,8 +7,12 @@
       "$ref": "./schema@v1.1.1#/definitions/basicProperties"
     }
   ],
-  "required": ["tables", "version", "status"],
+  "required": ["tables", "version", "status", "path"],
   "properties": {
+    "path": {
+      "type": "string",
+      "format": "uri"
```

Comment:
According to the JSON schema spec, a uri field must conform to [RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986), so they have to be full URIs. The paths are only URI fragments.